tianshou/policy/modelfree/bdq.py

```diff
@@ -57,9 +57,7 @@ def _target_q(self, buffer: ReplayBuffer, indices: np.ndarray) -> torch.Tensor:
             target_q = result.logits
         if self._is_double:
             act = np.expand_dims(self(batch, input="obs_next").act, -1)
-            act = torch.from_numpy(act)
-            if not target_q.get_device() < 0:  # TODO better?
-                act = act.to(target_q.get_device())
+            act = to_torch_as(act, target_q)
```

Comment:
Unfortunately it give a type error when running the pep8 check.

Reply:
I'll fix it